openstef/feature_engineering/missing_values_transformer.py

```diff
@@ -37,11 +38,16 @@ def __init__(
                 Can be one of "mean", "median", "most_frequent", "constant" or None.
             fill_value: When strategy == "constant", fill_value is used to replace all
                 occurrences of missing_values.
-
+            no_fill_future_values_features: The features for which it does not make sense
+                to fill future values. Rows that contain trailing null values for these
+                features will be removed from the data.
         """
         self.missing_values = missing_values
         self.imputation_strategy = imputation_strategy
         self.fill_value = fill_value
+        if no_fill_future_values_features is None:
+            no_fill_future_values_features = []
+        self.no_fill_future_values_features = no_fill_future_values_features
```

Comment:
A small nitpick, there is a shorthand for this.
```suggestion
        self.no_fill_future_values_features = no_fill_future_values_features or []
```